components/segmenter/src/rule_segmenter.rs

```diff
@@ -274,19 +280,12 @@ pub struct RuleBreakTypeUtf8;
 
 impl crate::private::Sealed for RuleBreakTypeUtf8 {}
 
-impl<'s> RuleBreakType<'s> for RuleBreakTypeUtf8 {
-    type IterAttr = CharIndices<'s>;
+impl RuleBreakType for RuleBreakTypeUtf8 {
```

Comment:
Suggestion: name the marker just `Utf8` instead of `RuleBreakTypeUtf8` so that you end up with something like `LineBreakIterator<Utf8>`

Reply:
I'll do that.